coral/src/app/features/team-info/TeamInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Typography } from "@aivenio/aquarium";
+import { useAuthContext } from "src/app/context-provider/AuthProvider";
+import { SwitchTeamsDropdown } from "src/app/features/team-info/SwitchTeamsDropdown";
+
+function TeamInfo() {
+  const authUser = useAuthContext();
+
+  function getTeamData() {
```

Comment:
Super nitpick: I think that functions that return components should be called `renderXX`, I was confused for a microsecond like "why do we get data in render :o"